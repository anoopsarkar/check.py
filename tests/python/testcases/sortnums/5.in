1231
453535
0
