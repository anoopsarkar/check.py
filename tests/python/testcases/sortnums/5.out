0
1231
453535
